commands/issue/update/issue_update.go

```diff
@@ -28,8 +28,33 @@ func NewCmdUpdate(f *cmdutils.Factory) *cobra.Command {
 		RunE: func(cmd *cobra.Command, args []string) error {
 			var err error
 			var actions []string
+			var assigneesToAdd, assigneesToRemove, assigneesToReplace []string
 			out := f.IO.StdOut
 
+			// Parse assignees Early so we can fail early in case of conflicts
+			if cmd.Flags().Changed("assignee") {
+				givenAssignees, err := cmd.Flags().GetStringArray("assignee")
```

Comment:
Should be read as a stringslice now
```suggestion
				givenAssignees, err := cmd.Flags().GetStringSlice("assignee")
```

Reply:
fixed, ty